### PR TITLE
Code cleanup, add models to ansible

### PR DESCRIPTION
**Pull request by hdefazio**

### CODE REVIEW

File: ansible/roles/provision-files/vars/main.yml [hunk 1]
```diff
@@ -45,12 +47,90 @@ girder_file_downloads:
...
+      sha512: 7ded5cb71ef0efe2444a19c7452073c63a34d65e3038b24236b304836abd47e8911e3d3023d2ba7e5a8d8c1e337e6f4b8e3080709827eb2b0daaedb0b6248561
+      dest: "{{ stage_dirs.pose_estimation }}/pose_model.pth"
+    # Hand detector
+    - file_id: 6605ca2e8b763ca20ae99f77
+      sha512: 42b56f34c0c443ad00d71e66334d6e852a7811011b22a60da9b5e2721565c21d1b5daad17c8a708908c9658ded81a663729090526feb5286bd0a591cf4fb5d8f
+      dest: "{{ stage_dirs.object_detector }}/hands_model.pt"
+    
+    # ---- M2 ----
+    # Object detector
+    # TODO
```
Comment (hdefazio) [inline ansible/roles/provision-files/vars/main.yml]: @periakiva  What object detector is used for M2?

**Comment by hdefazio** [comment]
@periakiva Can you merge this?

File: ansible/roles/provision-files/vars/main.yml (hunk 1 above)
Comment (periakiva) [reply ansible/roles/provision-files/vars/main.yml]: uploaded it to data.kitware.com

File: angel_system/global_step_prediction/medical_domain_experiment.py [hunk 2]
```diff
@@ -87,73 +83,15 @@ def get_unique(activity_ids):
...
 
     config = {
-        "m2": "/home/local/KHQ/peri.akiva/projects/angel_system/config/tasks/m2.yaml",
-        "m3": "/home/local/KHQ/peri.akiva/projects/angel_system/config/tasks/m3.yaml",
-        "m5": "/home/local/KHQ/peri.akiva/projects/angel_system/config/tasks/m5.yaml",
-        "r18": "/home/local/KHQ/peri.akiva/projects/angel_system/config/tasks/r18.yaml",
+        "m2": "config/medical/tasks/m2.yaml",
+        "m3": "config/medical/tasks/m3.yaml",
+        "m5": "config/medical/tasks/m5.yaml",
+        "r18": "config/medical/tasks/r18.yaml",
```
Comment (hdefazio) [inline angel_system/global_step_prediction/medical_domain_experiment.py:94]: @cameron-a-johnson

**Pull request merged**